### PR TITLE
Update release scripts to use python3

### PR DESCRIPTION
**Pull request by qiwzhang**
To fix b/195400955

### CODE REVIEW

**Comment by TAOXUY** [comment]
Hi @shuoyang2016, I remember you introduce python2 in the [Release Produce](http://go/esp-v2-release#release-procedure) step3. So you may take a look at this PR?

So Wayne, please update that doc if this PR checked in.

File: scripts/release/validate_release.py [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
 
+from absl import flags
 import collections
 import fnmatch
-import gflags
 import json
 import logging
 import os
-import prettytable
+from prettytable import PrettyTable
```
Comment (shuoyang2016) [inline scripts/release/validate_release.py:23]: Why this change?

**Review by shuoyang2016 — APPROVED** [review]

**Comment by shuoyang2016** [comment]
> Hi @shuoyang2016, I remember you introduce python2 in the [Release Produce](http://go/esp-v2-release#release-procedure) step3. So you may take a look at this PR?
> 
> So Wayne, please update that doc if this PR checked in.

I did. But I actually just renamed python to python2 :)

File: scripts/release/validate_release.py (hunk 1 above)
Comment (qiwzhang) [reply scripts/release/validate_release.py:23]: python3 package name changed to PrettyTable.
